app/js/index.js

```diff
@@ -0,0 +1,19 @@
+import React from 'react';
+import ReactDOM from 'react-dom';
+import { ConnectedRouter } from "connected-react-router";
+import { Provider } from 'react-redux';
+
+import 'bootstrap/dist/css/bootstrap.css';
```

Comment:
This is just a personal preference, I stay away from global CSS as it becomes harder to manage conflicts

Reply:
@iurimatias suggested me to use bootstrap. hence why it is there with Reactstrap.

I would like to try https://www.styled-components.com.
Would that align more with your preference?
That would mean more control and a long term win but also not as easy to start.

@iurimatias what do you think?

Reply:
styled-components are definitely aligned with this thinking.